RecoTracker/FinalTrackSelectors/BuildFile.xml

```diff
@@ -1,5 +1,4 @@
 <use name="ofast-flag"/>
-<use name="DataFormats/GeometryCommonDetAlgo"/>
```

Comment:
Indeed this was my mistake it seems - it should instead be in plugins/BuildFile.xml

Reply:
Okay, thanks for spotting this! I guess there will be more comments so I won't fix this immediately, but I keep it in mind for the next commit.